Symfony/CS/Tests/Fixer/Symfony/UnaryOperatorsSpacesFixerTest.php

```diff
@@ -84,8 +84,8 @@ public function provideCases()
                 '<?php function foo(& $a, array & $b, Bar & $c) {}',
             ),
             array(
-                '<?php foo(+$a, -2,-$b, &$c);',
-                '<?php foo(+ $a, - 2,- $b, & $c);',
```

Comment:
How do we want to handle test cases that are only valid in lower php versions?
Only test them in old version?
Test them in all versions? Maybe the best, because it could be possible that php 5.3 code is fixed with php 7. But then we could not activate linter in php7 builds on travis. 
Maybe add a flag to makeTest to deactivate linting for specific test cases?